featurebyte/routes/catalog/api.py

```diff
@@ -57,7 +58,7 @@ def __init__(self) -> None:
             status_code=HTTPStatus.OK,
         )
 
-        # update online store route
+        # update online store route (for backward compatibility; to be deprecated)
```

Comment:
I don't think we need to keep this? The update online store function is not in the latest released SDK

Reply:
This is mainly not to break the app frontend. Or is this route not use there yet?

Reply:
I see that makes sense 👍